scripts/gofmtcheck.sh

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
 # Check gofmt
-echo "==> Checking that code complies with gofmt requirements..."
```

Comment:
Curious why this line was removed? I find it helpful to see what a script is doing when it runs.

Reply:
I removed it because I wanted to use eval with make and keep this as a dependency (and this can't be eval'd). I also noticed this script is a dependency for every make rule so it was being noisy without doing anything if you run a few rules in succession.